package.json

```diff
@@ -66,7 +66,7 @@
     "test": "standard && npm run test-browser",
     "test-browser": "airtap --coverage --concurrency 1 -- test/*.js",
     "test-browser-local": "airtap --coverage --preset local -- test/*.js",
-    "test-node": "WRTC=wrtc tape test/*.js",
+    "test-node": "set WRTC=wrtc && tape test/*.js",
```

Comment:
?

Reply:
Most common format is as it was

Reply:
this was for testing, I've left it, as I couldn't bother finishing the PR, after wasting a solid 11 hours trying to fix tests, someone else will have to pick up this PR